README.md

```diff
@@ -1,18 +1,152 @@
-# Deploy HPCC Systems Platform with Terraform
-
-This set of Terraform examples deploys all the available features that come with the HPCC Systems OSS Terraform modules.
-
-## Order of deployment
-| Order | Name      | Required |
-| ----- | --------- | :------: |
-| 1     | `VNet`    |   yes    |
-| 2     | `AKS`     |   yes    |
-| 3     | `Storage` |    no    |
-| 4     | `Logging` |    no    |
-| 5     | `AKS`     |    no    |
-| 6     | `HPCC`    |   yes    |
-
-## Modules
-|                Name                 | Source | Used in |
-| :---------------------------------: | :----: | :-----: |
-| `terraform-azurerm-virtual-network` |        | `VNet`  |
\ No newline at end of file
+# Deploy HPCC Systems on Azure under Kubernetes
+
+NOTE: A tutorial of this Terraform for the developer, or others who are interested, can be found [here](documentation/hpcc-tf-for-developers.md).
+
+This is a slightly-opinionated Terraform module for deploying an HPCC Systems cluster on Azure's kubernetes service (aks).  The goal is to provide a simple method for deploying a cluster from scratch, with only the most important options to consider.
```

Comment:
Minor: 'kubernetes' should be capitalized.  Ensure capitalization in all external documentation.

Reply:
fixed